code/game/objects/items/stacks/stack_recipe.dm

```diff
@@ -17,10 +17,17 @@
 	var/time = 0
 	/// If only one of the resulting atom is allowed per turf
 	var/one_per_turf = FALSE
+	/// If the atom is fulltile, as in a fulltile window. This is used for the direction check to prevent fulltile windows from being able to be built over directional stuff.
+	/// Setting this to true will also set check_direction to true.
+	var/is_fulltile = FALSE
```

Comment:
should say something like "will effectively set X var to true". ideal world is that separate vars should stay separate, but i understand if the construction backend is too complicated to really allow something like this to not happen in some way.